### ISSUE TITLE
Replace PULP cluster DMA

### ISSUE THREAD

**Issue by adimauro-iis** [pulp-platform/pulp_cluster#81]
- [ ] Integrate the iDMA
- [ ] Write tests

**Comment by FrancescoConti** [comment]
@arpansur @da-gazzi we can move the technical discussion here maybe... and link any PR e.g., in HCI for the multi-initiatior arbiter.

**Comment by FrancescoConti** [comment]
Also.. @adimauro-iis if an issue actually involves more than one repo, do we capture that here?

**Comment by adimauro-iis** [comment]
Good point, I like a lot the idea of associating an issues to specific code changes into a repo. 

I think that whenever an issue requires changes into another repo we could:
1. Consider those changes as a pre-requisite for the issue, and assume they will be handled separately
2. Track them here, i.e. open an issue in the new repo and link it here as part of the task list, making them a sort of "sub-issue" 
> - [ ]   Let's pretend this issue https://github.com/pulp-platform/hci/issues/42 is needed for our feature to be added

Option 2 tracks everything but require more diligence...I am fine with both
